LeaderboardBackend/openapi.json

```diff
@@ -193,6 +193,49 @@
         }
       }
     },
+    "/Account/recover": {
+      "post": {
+        "tags": [
+          "Account"
+        ],
+        "summary": "Sends an account recovery email.",
+        "requestBody": {
+          "description": "The account recovery request.",
+          "content": {
+            "application/json": {
+              "schema": {
+                "$ref": "#/components/schemas/RecoverAccountRequest"
+              }
+            },
+            "text/json": {
+              "schema": {
+                "$ref": "#/components/schemas/RecoverAccountRequest"
+              }
+            },
+            "application/*+json": {
+              "schema": {
+                "$ref": "#/components/schemas/RecoverAccountRequest"
+              }
+            }
```

Comment:
Don't think these should be generated.

Reply:
I don't understand. The endpoint takes a JSON payload in the body.

Reply:
I fixed it anyway.